pkg/uptimerobot.go

```diff
@@ -112,6 +112,33 @@ func (a AlertContact) String() string {
 	return render(alertContactTemplate, a)
 }
 
+type Status int
```

Comment:
Doc comment please! (Actually, just hook in `golint` to your editor, and it will nag you about these before I can!)